native/src/routes/pois/containers/PoisContainer.js

```diff
@@ -133,16 +132,9 @@ const refresh = (refreshProps: RefreshPropsType, dispatch: Dispatch<StoreActionT
   navigateToPoi({ cityCode, language, path, forceRefresh: true, key: navigation.state.key })
 }
 
-type RestType = $Diff<PropsType, OwnPropsType>
-const removeOwnProps = (props: PropsType): RestType => {
-  const { t, navigation, ...rest } = props
-  return rest
-}
-
 export default withRouteCleaner<{| navigation: NavigationScreenProp<*> |}>(
-  withTranslation('pois')(
+  withTranslation('error')(

```

Comment:
Is this translation change correct? probably should be pois

Reply:
No, actually this is correct. The t function is just used for the SWITCH_LANGUAGE_ACTION to alert if no internet connection. I think that should be refactored though...